lib/postgrex/replication.ex

```diff
@@ -734,30 +606,12 @@ defmodule Postgrex.Replication do
     end
   end
 
-  @doc false
-  def handle_call({name, opts}, _from, %{repl_opts: nil, copy_opts: nil} = s)
-      when name in @public_commands do
-    statement = command(name, opts)
-
-    case Protocol.handle_simple(statement, s.protocol) do
-      {:ok, %Postgrex.Result{} = result, protocol} ->
-        {:reply, {:ok, result}, %{s | protocol: protocol}}
-
-      {:error, reason, protocol} ->
-        {:reply, {:error, reason}, %{s | protocol: protocol}}
-
-      {:disconnect, reason, protocol} ->
-        reconnect_or_stop(:disconnect, reason, protocol, s)
-    end
+  def handle_call({__MODULE__, _, _}, _from, s) do
+    {:reply, {:error, :stream_in_progress}, s}
   end
 
-  @doc false
-  def handle_call({name, _opts}, _from, s) when name in @public_commands,
-    do: {:reply, {:error, :stream_in_progress}, s}
-
-  @doc false
   def handle_call(msg, from, %{state: {mod, mod_state}} = s) do
-    handle(mod, :handle_call, [msg, from, mod_state], s)
+    handle(mod, :handle_call, [msg, from, mod_state], s, from)
```

Comment:
It seems strange to have `from` as the last argument, as it's usually the `state`. What about swapping them?

```suggestion
    handle(mod, :handle_call, [msg, from, mod_state], from, s)
```